test/plugins/features/movies/controller.test.js

```diff
@@ -11,8 +11,7 @@ describe('movie controller', () => {
 
       return Controller.create(payload)
       .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
-        expect(movie.get('name')).to.eql(payload.title);
+        expect(movie.get('name')).to.eql(payload.name);
```

Comment:
should this be `expect(movie.get('name')).to.eql(payload.title)`?

you're not passing in `name` in the payload, you're still passing in `title`. it's just being saved as `name`.